src/app/hierarchical-grid-updating/hierarchical-grid-updating.sample.ts

```diff
@@ -70,6 +70,14 @@ export class HierarchicalGridUpdatingSampleComponent implements AfterViewInit {
 
     gridCreated(event: IGridCreatedEventArgs, rowIsland: IgxRowIslandComponent) {
         this.remoteService.getData({ parentID: event.parentID, level: rowIsland.level, key: rowIsland.key }, (data) => {
+            if (rowIsland.key === 'Orders') {
```

Comment:
@kdinev, we can't use a pipe for this because the child grids don't have data properties and their data is set on application level through an event subscription.